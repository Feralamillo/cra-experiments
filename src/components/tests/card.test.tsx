import React from 'react';
import { shallow } from 'enzyme';

import { Card } from '../card';

test('renders the component', () => {
  const wrapper = shallow(<Card />);

  expect(wrapper).toMatchSnapshot();
});
