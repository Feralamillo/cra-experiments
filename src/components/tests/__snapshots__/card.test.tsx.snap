// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="col-md-4"
>
  <div
    className="card mb-4 box-shadow"
  >
    <img
      alt="Card"
      className="card-img-top"
      src="http://picsum.photos/200/200"
    />
    <div
      className="card-body"
    >
      <p
        className="card-text"
      >
        This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.
      </p>
      <div
        className="d-flex justify-content-between align-items-center"
      >
        <div
          className="btn-group"
        >
          <button
            className="btn btn-sm btn-outline-primary"
            type="button"
          >
            View
          </button>
          <button
            className="btn btn-sm btn-outline-secondary"
            type="button"
          >
            Edit
          </button>
        </div>
        <small
          className="text-muted"
        >
          9 mins
        </small>
      </div>
    </div>
  </div>
</div>
`;
